account_invoice_inter_company/models/res_config_settings.py

```diff
@@ -30,6 +30,10 @@ class ResConfigSettings(models.TransientModel):
         "(product where company is defined). Product not related to a "
         "company are visible for all companies.",
     )
+    module_account_invoice_inter_company_queued = fields.Boolean(
```

Comment:
Please don't do that. When migrating modules, this option won't be available until the other module is migrated, and confuse users.